jnigen/lib/src/elements/elements.dart

```diff
@@ -459,6 +462,7 @@ class Method extends ClassMember implements Element<Method> {
   final List<TypeParam> typeParams;
   List<Param> params;
   final TypeUsage returnType;
+  final String? descriptor;
```

Comment:
Maybe add documentation that this a unique descriptor within the class that can be used to match up with `KotlinFunction.descriptor`.